rp2040-hal/src/gpio/pin.rs

```diff
@@ -517,12 +520,36 @@ where
         self.regs.read_slew_rate()
     }
 
-    /// Set the slew rate for the pin
+    /// Set the slew rate for the pin.
     #[inline]
     pub fn set_slew_rate(&mut self, rate: OutputSlewRate) {
         self.regs.write_slew_rate(rate)
     }
 
+    /// Set the interrupt override.
+    #[inline]
+    pub fn set_interrupt_override(&mut self, r#override: InterruptOverride) {
+        self.regs.set_interrupt_override(r#override);
```

Comment:
can we avoid the raw label on the argument? maybe something like override_value or over_ride instead.

Reply:
I changed these to `override_value` now.